ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -609,17 +611,17 @@ function write(filter) {
             ' ' +
             filter.type +
             ' ' +
-            filter.value.toString(dateTimeFormat)
+            (filter.value ? filter.value.toString(dateTimeFormat) : "''")
           )
         case 'DURING':
           return (
             wrap(filter.property) +
             ' ' +
             filter.type +
             ' ' +
-            filter.from.toString(dateTimeFormat) +
+            (filter.from ? filter.from.toString(dateTimeFormat) : "''") +
```

Comment:
Should the pair of quotes be here? In other words, should this be:
```suggestion
            (filter.from ? filter.from.toString(dateTimeFormat) : "") +
```